L1Trigger/L1THGCalUtilities/plugins/ntuples/HGCalTriggerNtupleHGCDigis.cc

```diff
@@ -106,12 +117,15 @@ void HGCalTriggerNtupleHGCDigis::initialize(TTree& tree,
   tree.Branch("hgcdigi_eta", &hgcdigi_eta_);
   tree.Branch("hgcdigi_phi", &hgcdigi_phi_);
   tree.Branch("hgcdigi_z", &hgcdigi_z_);
-  char branchname[50];
+  std::string bname;
+  auto withBX([&bname](char const* vname, int bx) -> char const* {
```

Comment:
I used `int bx` in my example because `digiBXselect_` was a vector of `int`. But now it should rather be `unsigned`